AdminGUI/scripts/components/form-create.component.ts

```diff
@@ -1,8 +1,8 @@
 ﻿import { Component, OnInit, OnDestroy, NgZone, ViewChild, ElementRef } from '@angular/core';
 import { Router, ActivatedRoute } from '@angular/router';
-
 import { IEmptyConstruct, IEntityDataService, IDataStructure, NotificationService, GenericFormComponent, EntityChangeService, EntityWithType } from 'basecode/core';
```

Comment:
Should `NotificationService` be removed?

Reply:
Yes, it should.